lib/Bui.js

```diff
@@ -140,6 +142,27 @@ class Bui extends PureComponent {
             <br />
             <xmp style={styles.xmp}>{strongCode}</xmp>
           </div>
+          <div style={styles.componentContainer}>
```

Comment:
Could we add a header to match with the others? 
`<Header type="h3">Condensed Text Component</Header>`